mlflow/pyfunc/cli.py

```diff
@@ -1,13 +1,54 @@
 from __future__ import absolute_import
 
 import sys
+import os
+import subprocess
 
 import click
 import pandas
 
-from mlflow.pyfunc import load_pyfunc, scoring_server
+from mlflow.pyfunc import load_pyfunc, scoring_server, load_model_env
 from mlflow.tracking import _get_model_log_dir
 from mlflow.utils import cli_args
+from mlflow.projects import _maybe_create_conda_env, _get_conda_env_name
+import six
+
+
+def _rerun_in_conda(conda_env_path):
+    """ Rerun CLI command inside a to-be-created conda environment."""
+    conda_env_name = _get_conda_env_name(conda_env_path)
+    _maybe_create_conda_env(conda_env_path)
+    commands = []
+    commands.append("source activate {}".format(conda_env_name))
```

Comment:
This is subtle, but in theory we should find `activate` via `mlflow.projects._get_conda_bin_executable("activate")`. Maybe we could add a method in `mlflow.projects` called `_get_conda_activate_command(conda_env_name)`, which returns this string with the `source` part, so that we can share that with the code in
https://github.com/mlflow/mlflow/blob/d0841d6ed63bf67a7fc865686beb86d01f67ab3a/mlflow/projects/__init__.py#L312-L313

Reply:
Btw, you might need to rebase to get those changes, they were recent.

Reply:
got it! revised accordingly.